test/Table_Tests/src/Util.enso

```diff
@@ -47,6 +32,46 @@ DB_Column.should_equal self expected frames_to_skip=0 =
     t1 = expected.read
     t0 . should_equal t1 frames_to_skip
 
+## PRIVATE
+Table.should_equal_impl self expected loc =
```

Comment:
I'd prefer `should_equal_impl` to not be an extension method but a standalone method; this way we are not 'polluting' the namespace of methods defined on `Table`.